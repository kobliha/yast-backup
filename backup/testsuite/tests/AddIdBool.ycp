// Backup module - testsuite
//
// AddIdBool function tests
//
// testedfiles: backup/functions.ycp
//
// $Id$
//

{
import "Testsuite";

map readmap = $[ "target" : $["string" : "", "size" : -1, "tmpdir" : "/tmp" ],
    "proc" : $["cpuinfo" : $[ "value" : $["0" : $[ "flags" : ""]]]],
    "probe" : $["has_smp" : false, "is_uml" : false, "cdrom" : $["manual" : []]],
    ];

map execmap = $[ "target" : $["bash_output" : $["exit":0, "stderr":"", "stdout":"charmap=\"ISO-8859-1\"\n"]]];

Testsuite::Init([ readmap, $[], execmap ], nil);

include "backup/functions.ycp";

Testsuite::Dump( AddIdBool(nil, nil) );
Testsuite::Dump( AddIdBool([], true) );
Testsuite::Dump( AddIdBool([ `item(`id("ext2"), "ext2", true)], false) );
Testsuite::Dump( AddIdBool(["abcd"], true) );

}

