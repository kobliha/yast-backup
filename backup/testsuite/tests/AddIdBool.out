Dump	[]
Dump	[]
Dump	[`item (`id ("ext2"), "ext2", false)]
Dump	[nil]
